<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordManagment</class>
 <widget class="QDialog" name="PasswordManagment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Changer son mot de passe</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="old_password_label">
       <property name="text">
        <string notr="true">Ancien mot de passe:</string>
       </property>
       <property name="buddy">
        <cstring>old_password_input</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Input" name="old_password_input">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="Input" name="new_password_input">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="pseudo_label">
       <property name="text">
        <string notr="true">Nom d'utilisateur:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="new_password_label">
       <property name="text">
        <string notr="true">Nouveau mot de passe:</string>
       </property>
       <property name="buddy">
        <cstring>new_password_input</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="pseudo_input"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string notr="true">Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="accept_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">Changer</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Input</class>
   <extends>QLineEdit</extends>
   <header>gui.input_widget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>accept_button</sender>
   <signal>clicked()</signal>
   <receiver>PasswordManagment</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>PasswordManagment</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
